From a96e5a62463adae4c97de1e842c3287d39345792 Mon Sep 17 00:00:00 2001
From: Emilian Bold <emi@apache.org>
Date: Tue, 11 Dec 2018 15:54:59 +0200
Subject: [PATCH 23/32] Fixes the 8.2 Plugin Portal URL and sets it to the
 certified plugins
License: Copyright (c) 2018, 2019 Emilian Marius Bold
 
 Permission to use, copy, modify, and distribute this software for any
 purpose with or without fee is hereby granted, provided that the above
 copyright notice and this permission notice appear in all copies.
 
 THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
 WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
 MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
 ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
 WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
 ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
 OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.

---
 .../netbeans/modules/updatecenters/resources/Bundle.properties  | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/nb/updatecenters/src/org/netbeans/modules/updatecenters/resources/Bundle.properties b/nb/updatecenters/src/org/netbeans/modules/updatecenters/resources/Bundle.properties
index 3c178c9d94..b076297e0b 100644
--- a/nb/updatecenters/src/org/netbeans/modules/updatecenters/resources/Bundle.properties
+++ b/nb/updatecenters/src/org/netbeans/modules/updatecenters/resources/Bundle.properties
@@ -32,7 +32,7 @@ URL_Distribution=@@metabuild.DistributionURL@@
 #NOI18N
 URL_PluginPortal=@@metabuild.PluginPortalURL@@
 #NOI18N
-URL_82PluginPortal=http://updates.netbeans.org/netbeans/updates/8.2/uc/final/distribution/catalog.xml.gz
+URL_82PluginPortal=http://updates.netbeans.org/netbeans/updates/8.2/uc/final/certified/catalog.xml.gz
 
 3rdparty=Third Party Libraries
 #NOI18N
-- 
2.21.0

